index.bs

```diff
@@ -1208,7 +1208,8 @@ run the following steps:
     Note: The exact value of |expires| is not important for the purposes of this algorithm,
     as long as it is in the past.
 
-1. Let |value| be any non-empty [=implementation-defined=] string.
+1. If |name|'s [=string/length=] is 0, let |value| be any non-empty [=implementation-defined=] string.
+1. Otherwise, let |value| be the empty string. 
```

Comment:
Usually this is the preferred style:
```suggestion
1. Let |value| be the empty string. 
1. If |name|'s [=string/length=] is 0, then set |value| to any non-empty [=implementation-defined=] string.
```
Or alternatively:
```suggestion
Let |value| be the empty string if |name| is not the empty string; otherwise any non-empty [=implementation-defined=] string.
```